src/viewer/modmesh/viewer/PythonInterpreter.cpp

```diff
@@ -134,9 +134,12 @@ PYBIND11_EMBEDDED_MODULE(_modmesh_view, mod)
             {
                 RApplication * app = RApplication::instance();
                 RScene * scene = app->main()->viewer()->scene();
-                for (RStaticMesh<2> * child : scene->findChildren<RStaticMesh<2> *>())
```

Comment:
The `child` includes the camera controller which gets deleted although it shouldn't be.